### PR TITLE
1.5 QA bug fixes

### PR DESCRIPTION
**Pull request by Meldiron**
## What does this PR do?

- Removes no longer needed translations
- Fixes unwanted empty spaces in email copy about client device
- Blocks email otp if disabled on project
- Fixes nested templates and problematic plaintext mails
- Fixes unescaped HTML in magic URL & email otp mails

## Test Plan

- [x] Manual QA

## Related PRs and Issues

x

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: src/Appwrite/Platform/Workers/Mails.php [hunk 1]
```diff
@@ -78,8 +78,9 @@ public function action(Message $message, Registry $register, Log $log): void
         $bodyTemplate = Template::fromFile($bodyTemplate);
         $bodyTemplate->setParam('{{body}}', $body, escapeHtml: false);
         foreach ($variables as $key => $value) {
-            // TODO: hotfix for redirect param
-            $bodyTemplate->setParam('{{' . $key . '}}', $value, escapeHtml: $key !== 'redirect');
+            // TODO: hotfix for html params
+            $htmlParams = [ 'redirect', 'agentDevice', 'agentClient', 'agentOs', 'phrase' ];
```
Comment (TorstenDittmann) [inline src/Appwrite/Platform/Workers/Mails.php:82]: why cant those be escaped?

I think all of those expect redirect should be scaped.

Comment (Meldiron) [reply src/Appwrite/Platform/Workers/Mails.php:82]: They have boldness, so `<b>SOMETHING</b>`. Without it the email looks very plain, and I couldn't find better way to do it

Comment (Meldiron) [reply src/Appwrite/Platform/Workers/Mails.php:82]: This is what it looks likewithout changes in my PR
CleanShot 2024-01-31 at 10 05 57@2x

File: app/controllers/shared/api/auth.php [hunk 2]
```diff
@@ -31,6 +31,12 @@
                 }
                 break;
 
+            case 'emailOtp':
```
Comment (TorstenDittmann) [inline app/controllers/shared/api/auth.php:34]: can we call it `email-otp` so it aligns with the other values like `magic-url`?

**Comment by stnguyen90** [comment]
@Meldiron, is this PR still needed?

**Comment by stnguyen90** [comment]
I'm closing this stale issue. Please create issues to track anything that still needs to be done.

**Pull request closed without merging**